test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/005_OnVehicleData_notification_ignored.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: SDL does not send OnVehicleData notification to mobile app if it notification is invalid
+-- In case:
+-- 1) App is subscribed to get `FuelRange` data
+-- 2) HMI sends invalid OnVehicleData notification of FuelRange structure:
+-- (type, range, level, levelState, capacity, capacityUnit)
+-- SDL does:
+-- 1) ignore this notification
+-- 2) not send OnVehicleData notification to mobile
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/common')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local params = {
+  fuelRange = {
+    dataType = "VEHICLEDATA_FUELRANGE",
+    resultCode = "SUCCESS"
+  }
+}
+
+local function onVehicleData(pParams)
```

Comment:
Function common.sendOnVehicleData can be updated with new parameter that is define the number of expected OnVD notifications